service/worker/esanalyzer/domainWorkflowTypeCountWorkflow.go

```diff
@@ -160,6 +165,80 @@ func (w *Workflow) emitWorkflowTypeCountMetrics(ctx context.Context) error {
 	return nil
 }
 
+func (w *Workflow) getDomainWorkflowTypeCountPinotQuery(domainName string) (string, error) {
+	domain, err := w.analyzer.domainCache.GetDomain(domainName)
+	if err != nil {
+		return "", err
+	}
+	// exclude uninitialized workflow executions by checking whether record has start time field
+	return fmt.Sprintf(`
+SELECT WorkflowType, COUNT(*) AS count
+FROM %s
+WHERE DomainID = '%s'
+AND IsDeleted = false
+AND CloseStatus = -1
+AND StartTime > 0
+GROUP BY WorkflowType
+ORDER BY count
+LIMIT 10
```

Comment:
ES search returns at most top 10 aggregation results. I don't see ORDER by here in pinot query